### PR TITLE
Added question enter destination folder path

### PR DESCRIPTION
**Pull request by manjeetbeniwalinnostax**
Description: Added new question for Destination folder path, provide new path for project template creation.

### CODE REVIEW

File: src/generateRoutes/generate-route.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 const inquirer = require("inquirer");
 const fs = require("fs");
 const fsExtra = require("fs-extra");
-const CURR_DIR = process.cwd();
+// const CURR_DIR = process.cwd();
 const path = require("path");
```
Comment (manjeetbeniwalinnostax) [inline src/generateRoutes/generate-route.js:6]: Will remove this comment in next PR

File: src/generateRoutes/generate-route.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ const QUESTIONS = [
   },
 ];
 inquirer.prompt(QUESTIONS).then((answers) => {
+  const CURR_DIR = answers["projectDirectoryPath"] ? answers["projectDirectoryPath"] : process.cwd();
```
Comment (Chaman-Raghav-Inno) [inline src/generateRoutes/generate-route.js:23]: answers["projectDirectoryPath"] ? answers["projectDirectoryPath"] : process.cwd(); 
=>
answers["projectDirectoryPath"] || process.cwd();

File: src/questionnaire.js [hunk 3]
```diff
@@ -9,6 +9,14 @@ module.exports = [
...
     },
   },
+  {
+    name: "projectDirectoryPath",
+    type: "input",
+    message: "Enter destination folder path?",
+    when: (answers) => {
+      return answers.projectName;
+    },
+  },
```
Comment (Chaman-Raghav-Inno) [inline src/questionnaire.js:19]: do we have a fallback, if entered path is not available in system? or someone miss typed or pasted wrong path, will he get errors?

File: src/utils/helper.js [hunk 4]
```diff
@@ -23,8 +22,10 @@ function createDirectoryContents(
   nodeName,
   projectChoice,
   isThemeProvider,
-  isMaterialUI
+  isMaterialUI,
+  currentDirectory
 ) {
+  const CURR_DIR = currentDirectory ? currentDirectory : process.cwd();
```
Comment (Chaman-Raghav-Inno) [inline src/utils/helper.js:28]: currentDirectory ? currentDirectory : process.cwd();
=>
currentDirectory || process.cwd();
if we can update, let's check in whole PR

**Pull request merged**